tests/integration/solidity_contracts/Solmate/ERC721.sol

```diff
@@ -147,7 +147,7 @@ contract ERC721 {
         );
     }
 
-    function safeTransferFrom(
+    function safeTransferFrom2(
```

Comment:
Changed to avoid ambiguity in method names, otherwise was seeing this error:

   ```
    def get_function_by_identifier(
        fns: Sequence[ContractFunction], identifier: str
    ) -> ContractFunction:
        if len(fns) > 1:
           raise ValueError(
                'Found multiple functions with matching {0}. '
                'Found: {1!r}'.format(identifier, fns)
              ValueError: Found multiple functions with matching name. Found: [<Function safeMint(address,uint256,bytes)>, <Function safeMint(address,uint256)>
  ```